remove.go

```diff
@@ -10,5 +12,6 @@ func removeCommand(args ...string) int {
 		return handleError(err)
 	}
 
+	fmt.Println("Credential was removed")
```

Comment:
The `add` command prints `Credential added`, so let's go with `Credential removed` here for consistency.